src/components/NestedTabs/NestedTabs.tsx

```diff
@@ -0,0 +1,13 @@
+import { Tabs } from '@patternfly/react-core';
+import { PropsWithChildren } from 'react';
+
+interface INestedTabsProps {
+  activeTabKey: number;
+  onTabSelect: (tabKey: INestedTabsProps['activeTabKey']) => void;
+}
+
+export const NestedTabs = ({ activeTabKey, onTabSelect, children }: PropsWithChildren<INestedTabsProps>) => (
```

Comment:
Add also TSDoc. It has to include also its recommended usage - it can be used only in very specific cases (when switch-like functionality is sufficient) as we require all pages to be reachable via link, but in this case it's not fulfilled. For this purpose there is another jira, but we won't implement it now.